packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-tabs.tsx

```diff
@@ -1,28 +1,77 @@
-import { EntityType } from "@blockprotocol/type-system-web";
+import { extractBaseUri, VersionedUri } from "@blockprotocol/type-system-web";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system";
-import { Box, Typography } from "@mui/material";
+import { frontendUrl } from "@hashintel/hash-shared/environment";
+import { Box, Tabs, tabsClasses } from "@mui/material";
+import { useRouter } from "next/router";
+import { useEffect, useMemo, useState } from "react";
 import { EntityTypeDefinitionTab } from "./entity-type-definition-tab";
+import { EntityTypeEntitiesTab } from "./entity-type-entities-tab";
 import { TabButton } from "./tab-button";
+import { useEntityType } from "./use-entity-type";
+
+export const EntityTypeTabs = () => {
+  const router = useRouter();
+  const entityType = useEntityType();
+
+  const [activeTab, setActiveTab] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    setActiveTab(`${frontendUrl}${router.asPath}`);
+  }, [router.asPath]);
```

Comment:
can't you take the `activeTab` from `router.asPath` directly, rather than using state? 

You could also just make the tab value "entities" (i.e. the last bit of the URI) to save all combining URIs that is going on elsewhere

Reply:
Passing  `router.asPath` directly to the `value` of Tabs in 95df93c98b25355a322427b57d528d724d9a5781. 
This also makes it so the tabs animation is no longer decoupled from the page contents (it would animate first and then show the table after a small delay). Thank you for the suggestion.

Reply:
Now passing `router.query.tab` directly to the `value` of Tabs 6e77bb686f3563df662a711b8e95e8676defc7c8.